website/templates/project/addon/widget.mako

```diff
@@ -1,31 +1,33 @@
-<div class="panel panel-default" name="${short_name}">
+% if complete or 'write' in user['permissions']:
+    <div class="panel panel-default" name="${short_name}">
 
-            <div class="panel-heading clearfix">
-                <h3 class="panel-title">${full_name}</h3>
-                <div class="pull-right">
-                    % if has_page:
-                       <a href="${node['url']}${short_name}/">  <i class="fa fa-external-link"></i> </a>
-                   % endif
+                <div class="panel-heading clearfix">
```

Comment:
This looks too far indented. I think it should only be 8 spaces from the beginning of the line, yes?